ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/answers/activityNumericAnswer.component.ts

```diff
@@ -46,10 +46,20 @@ export class ActivityNumericAnswer implements OnInit, OnChanges, OnDestroy {
     public numericField: FormControl;
     InputRestriction = InputRestriction;
     private subs: Subscription;
+    public minValue: Number;
+    public maxValue: Number;
 
     public ngOnInit(): void {
         this.initForm();
-
+        const validatorItem=this.block.validators.find((item) => !!item);
```

Comment:
We have a separate mapper for transformation a DTO data to a front-end model (see https://github.com/broadinstitute/ddp-angular/pull/1602#issuecomment-1199200283).
No need for updating the component.

Reply:
After taking a look at  projects\ddp-sdk\src\lib\services\activity\activityQuestionConverter.service.ts I adjusted part of the logic related to the values for min and max.
Now we can see the input is taking the values properly:
![image](https://user-images.githubusercontent.com/109761417/181838732-e1e9b603-b5b9-423a-9d32-2220676342ab.png)